packages/trader/src/AppV2/Components/TradeParameters/Multiplier/multiplier-wheel-picker.tsx

```diff
@@ -68,7 +69,10 @@ const MultiplierWheelPicker = ({
                     <Text size='sm' as='div' className='multiplier__commission-value'>
                         {commission ? (
                             <React.Fragment>
-                                {commission} {currency}
+                                {FormatUtils.formatMoney(+commission, {
```

Comment:
will it be formatMoney from shared package or this one ? 
I mean I don't exactly remember , but there was some edge case due to which I added formatMoney (from shared packages) in useOrderDetails hook . have to see if FormatUtil.formatMoney cover the implementation for the formatMoney (shared package).

Reply:
`formatMoney` from `shared` seems to be deprecated. I followed the idea of 1) if in file was already imported `formatMoney` (shared package), I used it. 2) if in file there was no `formatMoney`, I imported it from `@deriv-com/utils`. 
In the code I saw the usage of both functions. I'm not sure about the edge cases, but I think we can have an agreement with team which one to use and then to refactore the code

<img width="695" alt="Screenshot 2024-09-23 at 10 32 00 AM" src="https://github.com/user-attachments/assets/3a111481-96de-4295-828e-3a33df7f1024">

Reply:
@ahmadtaimoor-deriv do you recommend to use `formatMoney` (from shared packages) because of edge case?

Reply:
yeah , just to be on safe side